src/ImageSharp/Image.WrapMemory.cs

```diff
@@ -150,5 +150,70 @@ public abstract partial class Image
             int height)
             where TPixel : unmanaged, IPixel<TPixel>
             => WrapMemory(Configuration.Default, pixelMemoryOwner, width, height);
+
+        /// <summary>
+        /// Wraps an existing contiguous memory area of 'width' x 'height' pixels,
+        /// allowing to view/manipulate it as an ImageSharp <see cref="Image{TPixel}"/> instance.
+        /// </summary>
+        /// <typeparam name="TPixel">The pixel type</typeparam>
+        /// <param name="configuration">The <see cref="Configuration"/></param>
+        /// <param name="byteMemory">The byte memory representing the pixel data.</param>
+        /// <param name="width">The width of the memory image.</param>
+        /// <param name="height">The height of the memory image.</param>
+        /// <param name="metadata">The <see cref="ImageMetadata"/>.</param>
+        /// <exception cref="ArgumentNullException">The configuration is null.</exception>
+        /// <exception cref="ArgumentNullException">The metadata is null.</exception>
+        /// <returns>An <see cref="Image{TPixel}"/> instance</returns>
+        public static Image<TPixel> WrapMemory<TPixel>(
```

Comment:
<https://docs.microsoft.com/en-us/dotnet/standard/memory-and-spans/memory-t-usage-guidelines>

Seems beneficial to consider exposing `IMemoryOwner<T>` overloads of this to allow ownership transfer?

Reply:
I didn't include them here on purpose as they weren't part of the initial API proposal in #1097.
@antonfirsov should this be added here or would that be for another PR?

Reply:
No preference, other than it's always easier to review multiple smaller (subsequent) PR-s.

Reply:
Sounds like a preference to me 😄
Will keep this PR restricted to consumed buffers then, and we can create another one after this one is merged to also add support for owned ones through `IMemoryOwner<byte>` then.